### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Dylanharo575** [lllyasviel/Fooocus#2804] [General]
Hello,
I was running the program for some time and had no issues. I ran it one day and I keep coming up with the following error. I've tried disabling the firewall, manually inputting DNS, adding registry entry, and the basic delete all files and reinstall anyone have any ideas? Also on my extracted files for (run) it's not showing (run.bat) like it used to.  ***fixed issue*** I disabled Ipv6 worked after

![fooocus error](https://github.com/lllyasviel/Fooocus/assets/154105996/5a35b005-364e-4c52-ad05-177bd0ff03b5)

**Comment by mashb1t** [comment]
We've received multiple reports today that HuggingFace might have had issues. Please check and try again.
